next-tavla/app/(admin)/organizations/components/UploadLogo/utils.ts

```diff
@@ -8,3 +8,7 @@ export function getFilename(logoUrl?: string) {
 
     return file[1]
 }
+export function containsSpecialChars(text: string) {
+    const specialChars = /[`!@#$%^&*()+\=\[\]{};':"\\|,<>\/?~]/
```

Comment:
Jeg tenkte på om det var lettere å sjekke motsatt vei, at man sjekker om filnavnet kun inneholder gyldige karakterer? Nå må vi huske å oppdatere om det kommer nye spesialtegn i fremtiden (f. eks emojies idk). Sjekker man for at kun bokstaver, tall +++ er lov, så er man sikret for alt annet.

Reply:
Veldig bra forslag 😄